app/src/features/apiClient/screens/apiClient/components/clientView/components/request/request-body-state-manager.ts

```diff
@@ -32,14 +38,35 @@ export class RequestBodyStateManager {
     return this.form;
   }
 
+  getMultiPartForm() {
+    return this.multiPartForm;
+  }
+
   serialize(): RQAPI.RequestBodyContainer {
     return {
       text: this.text,
       form: this.form,
+      multiPartForm: this.multiPartForm,
     };
   }
 }
 
+/**
+ * This is a utility hook which enables syncing react state and 'RequestBodyStateManager'
+ */
+export function useMultiPartFormBody(requestBodyStateManager: RequestBodyStateManager) {
```

Comment:
Test this well, make sure that data syncs two way
1. From state manager to component
2. From component to state manager